doc/samples/sid_end_device.rst

```diff
@@ -25,6 +25,8 @@ This sample supports the following development kits:
 +--------------------+----------+------------------------+------------------------------+
 | nRF5340 DK         | PCA10095 | `nrf5340dk_nrf5340`_   | ``nrf5340dk_nrf5340_cpuapp`` |
 +--------------------+----------+------------------------+------------------------------+
+| Thingy53           |          | `thingy53_nrf5340`_    | ``thingy53_nrf5340_cpuapp``  |

```

Comment:
PCA needs to be added as well.  Would it be PCA20053 ?

Reply:
PCA20053 added